tests/browser/levels/create_edit_delete_level.robot

```diff
@@ -6,17 +6,21 @@ Suite Teardown  Delete Records and Close Browser
 
 *** Variables ***
 ${level_name}
+${contact_id}
 
 *** Test Cases ***
 
-Create Level and Verify
+1 Create Level and Verify Fields
```

Comment:
Are the numbers needed in order to have a stable order that the tests run in, or was there another reason you added them?

Reply:
Actually I didn't realize the tests ran out of order. If that's the case, then yes. But the real reason was that the list of tests passed/failed in the results was alphabetical versus in the order that I had them in the robot file.

Reply:
I think they do run in the order in the file. Do you mean they are alphabetical in the results shown in MetaCI, or somewhere else?

Reply:
In the log.html as well as in the results page in MetaCI.

Reply:
That surprises me that it's not in the order it ran in log.html. It doesn't surprise me so much that it's out of order in MetaCI, but we probably can't fix that if they're out of order in the log. Anyway, I don't have a problem with the numbers; was just curious.

Reply:
You can see that in this report (at the very bottom). The sequence of the tests listed under the "Tests/Browser/Levels/Create Edit Delete Level" test do not match the sequence of the tests in the robot file (i.e, the "Delete Level and Validate Contact" test should be last).
https://mrbelvedereci.herokuapp.com/tests/42724/robot